contributed_definitions/nyaml/NXcs_cpu_obj.yaml

```diff
@@ -5,7 +5,7 @@ symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
 type: group
-NXcs_cpu_obj(NXobject):
+NXcs_cpu_obj(NXcircuit):
```

Comment:
Not convinced that this should be its own base class. Just add `name` and `NXfabrication` to `NXcircuit` and just `cs_gpu_obj(NXcircuit` when you want to talk about GPUs.

Reply:
Agree, followed your suggestion and refactored this set of base classes, only NXcs_computer remains using NXcircuit.